rxkata/src/main/java/org/sergiiz/rxkata/CountriesServiceSolved.java

```diff
@@ -80,4 +80,24 @@
                         country -> country.name,
                         country -> country.population);
     }
+
+    @Override
+    public Observable<Long> combineAndReturnTheSumOfPopulation(Observable<Country> countryObservable1,
+                                                               Observable<Country> countryObservable2) {
+        return Observable.merge(countryObservable1, countryObservable2)
+                .map(country -> country.population)
+                .reduce((i1, i2) -> i1 + i2)
+                .toObservable();
+    }
+
+    @Override
+    public Single<Boolean> checkIfBothObservableEmitsSameItems(Observable<Country> countryObservable1, Observable<Country> countryObservable2) {
+        return Observable.sequenceEqual(countryObservable1,countryObservable2);
+    }
+
+    @Override
+    public Single<Boolean> checkIfObservableEmitsGivenItem(List<Country> countries, Country country) {
+        return Observable.fromIterable(countries)
+                .contains(country);
```

Comment:
I'm wondering if there are any more elegant solution for this and if it has a practical use-case.